collector/generatorreceiver/internal/topology/metric.go

```diff
@@ -73,11 +80,24 @@ func (m *Metric) GetValue() float64 {
 			return SquareValue(phase)
 		case Triangle:
 			return TriangleValue(phase)
+		case Average:
+			return AverageValue(phase)
 		default:
 			// TODO: what would be a reasonable default? Maybe just sine?
 			return SineValue(phase)
 		}
 	}(phase)
 
-	return m.Min + (m.Max-m.Min)*factor
+	v := m.Min + (m.Max-m.Min)*factor
+
+	// jitter deviation is calculated in percentage that ranges from [-m.Jitter/2, m.Jitter/2)%
+	j := 1 + random.Float64()*m.Jitter - m.Jitter/2
```

Comment:
This would entail having a min and max jitter value too, which would be great - like, only positive jitter in particular I think would be good to support

Reply:
I like that, I'll keep that in mind for a future work.